api/v1/storagecluster_types.go

```diff
@@ -19,6 +19,7 @@ package v1
 import (
 	nbv1 "github.com/noobaa/noobaa-operator/v2/pkg/apis/noobaa/v1alpha1"
 	conditionsv1 "github.com/openshift/custom-resource-status/conditions/v1"
+	cephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
```

Comment:
Please please please name this `rookCephv1`.

Reply:
Updating for consistency. Kinda wish the whole thing was `cephv1` :laughing:

Reply:
Looks like `cephv1` is more common in the codebase than `rookCephv1`. This and a few files are an outlier! updating `rookCephv1` to `cephv1` in this file :)